azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/ClientEntity.java

```diff
@@ -101,17 +101,13 @@ public final void closeSync() throws EventHubException {
                 Thread.currentThread().interrupt();
             }
 
-            Throwable throwable = exception.getCause();
-            if (throwable != null) {
-                if (throwable instanceof RuntimeException) {
-                    throw (RuntimeException) throwable;
-                }
-
-                if (throwable instanceof EventHubException) {
-                    throw (EventHubException) throwable;
-                }
-
-                throw new EventHubException(true, throwable);
+            final Throwable throwable = exception.getCause();
+            if (throwable instanceof RuntimeException) {
+                throw (RuntimeException) throwable;
+            } else if (throwable instanceof EventHubException) {
+                throw (EventHubException) throwable;
+            } else {
+                throw new RuntimeException(throwable != null ? throwable : exception);
```

Comment:
This is change from previous behavior, is there any reason of wrapping it as RuntimeException?